components/accordion-menu/script.js

```diff
@@ -0,0 +1,157 @@
+import components from 'coherent-gameface-components';
+
+const PADDING_BOTTOM = 16;
+
+/**
+ *  @class AccordionMenu
+ * AccordionMenu component, allows you to create a menu with drawers that can be expanded or shrunk on click
+ */
+class AccordionMenu extends HTMLElement {
+    constructor() {
+        super();
+
+        this.panelClick = this.panelClick.bind(this);
+    }
+
+    connectedCallback() {
+        //Check if it has the multiple attribute
+        this._multiple = this.hasAttribute('multiple');
+
+        //Remove the border from the last panel
+        this.lastChild.classList.add('accordion-last-panel');
```

Comment:
```suggestion
        this.lastElementChild.classList.add('accordion-last-panel');
```
In order to work in the browser also.